pkg/kubelet/cri/remote/fake/fake_runtime.go

```diff
@@ -322,3 +322,18 @@ func (f *RemoteRuntime) ReopenContainerLog(ctx context.Context, req *kubeapi.Reo
 
 	return &kubeapi.ReopenContainerLogResponse{}, nil
 }
+
+// CheckpointContainer checkpoints the given container.
+func (f *RemoteRuntime) CheckpointContainer(ctx context.Context, req *kubeapi.CheckpointContainerRequest) (*kubeapi.CheckpointContainerResponse, error) {
+	err := f.RuntimeService.CheckpointContainer(req.Id, "checkpointDir")
+	if err != nil {
+		return nil, err
+	}
+
+	return &kubeapi.CheckpointContainerResponse{}, nil
+}
+
+// CheckpointContainer checkpoints the given container.
```

Comment:
Is this method needed for this PR? and if yes, the comment here needs to be updated to describe the restore

Reply:
Yes, it is needed because of the CRI API changes and it does not compile without this. Thanks for pointing out the wrong comment. Fixed.